docs/migration-guide-2.15-2.16.md

```diff
@@ -0,0 +1,8 @@
+
+---
+Migration Guide 2.15-2.16
+---
+
+Loki has been removed with Kyma 2.16. After upgrading to Kyma 2.16, run the script [2.15-2.16-cleanup-loki.sh](./assets/2.15-2.16-cleanup-loki.sh) to remove Loki.
```

Comment:
or 2.17?

Reply:
yes, it will be 2.17 now. I will change it